rust/datafusion/src/execution/context.rs

```diff
@@ -426,6 +430,51 @@ impl ExecutionContext {
             }
         }
     }
+
+    /// Execute a query and write the results to a partitioned CSV file
+    pub fn write_csv(&self, plan: &dyn ExecutionPlan, path: &str) -> Result<()> {
+        // create directory to contain the CSV files (one per partition)
+        let path = path.to_string();
+        println!("Creating {}", path);
```

Comment:
It might be worthwhile to implement some logging in datafusion to help users with diagnostics

Reply:
Good catch. Yes, there is a JIRA for adding logging.